src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,37 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+@import "../utils/functions";
+@import "../normalize/normalize.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: 7 * $cell-size + 10 * $cell-gap;
+
+  &__cell {
+    @include square ($cell-size);
+    background-color: #{$cell-color};
+    border: #{$cell-border};
+    display: flex;
+    justify-content: center;
```

Comment:
Organize you styles 
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)